.gitignore

```diff
@@ -8,5 +8,6 @@ npm-debug.log
 /test/config.js
 
 *.sublime-*
+.idea
```

Comment:
Please remove this; really, editor-specific entries belong in your global ignore list.